apps/tokenomics/common-util/functions/requests.js

```diff
@@ -0,0 +1,16 @@
+const ESTIMATED_GAS_LIMIT = 500_000;
+
+/**
+ * function to estimate gas limit
+ */
+export const getEstimatedGasLimit = async (fn, account) => {
+  try {
+    const estimatedGas = await fn.estimateGas({ from: account });
+    return Math.floor(estimatedGas);
```

Comment:
last time we agreed to set some extra limit so the tx not fail in case it's recalculated to the higher gas, do you think it make sense to fix it here as well? 

`return Math.ceil(estimatedGas * 1.2);`

see discussion: https://github.com/valory-xyz/autonolas-contribution-service-frontend/pull/134#discussion_r1567124867